lua/apg/cl_menu.lua

```diff
@@ -397,22 +398,31 @@ properties.Add( "apgoptions", {
 			icon = "icon16/tick.png",
 			callback = function() self:APGcmd( ent, "remghost" ) end,
 		})
+
+		submenu:AddSpacer()
+
+		addoption( "Ghost this entity", {
+			icon = "icon16/tick.png",
+			callback = function() self:APGcmd( ent, "ghost" ) end,
+		})
+
 	end,
 	Action = function( self, ent ) end,
 	APGcmd = function( self, ent, cmd )
 		if cmd == "getownerid" then
 			local owner, _ = ent:CPPIGetOwner()
-			if IsValid( owner ) and owner.SteamID() then
+			if IsValid( owner ) then

```

Comment:
Do we know why this check was there before? Could it have been guarding from an exception?